readtab.c:32:10: error: 'obstack.h' file not found with <angled> include; use "quotes" instead
--- readtab.c.orig	2000-06-22 17:13:12.000000000 -0500
+++ readtab.c	2021-02-14 04:19:56.000000000 -0600
@@ -29,7 +29,7 @@
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <obstack.h>
+#include "obstack.h"
 #include <limits.h>
 #include <fnmatch.h>
 #include <unistd.h>
